docs/raising-events.md

```diff
@@ -34,20 +34,33 @@ robot.FellInLove += Raise.WithEmpty(); // sender will be robot, args will be Eve
 robot.FellInLove += Raise.With(sender: robot, e: someEventArgs);
 ```
 
-Events of type `EventHandler<TEventArgs>` can be raised in exactly the same way. 
+Events of type `EventHandler<TEventArgs>` can be raised in exactly the same way.
 
-Note that as of version 3.1.0, for the .NET Standard version of the FakeItEasy library, `TEventArgs` need not extend `EventArgs` to support the above syntax. The .NET 4.0 library still requires `TEventArgs` to extend `EventArgs`.
+Note that as of version 3.1.0, for the .NET Standard and .NET 4.5 versions of the FakeItEasy library, `TEventArgs` need not extend `EventArgs` to support the above syntax. The .NET 4.0 library still requires `TEventArgs` to extend `EventArgs`.

```

Comment:
This is a good point. A holdover from when we had them in the wiki perhaps.
Of course, even when docs are versioned, sometimes it's nice to indicate when a feature became available or changed.

Still, FakeItEasy should be upgradeable on a whim, and the .NET Standard behaviour has been there for over 3 releases! I think you're right. We could just  drop the versions.